Core/include/Acts/Propagator/MultiEigenStepperLoop.hpp

```diff
@@ -466,7 +466,7 @@ class MultiEigenStepperLoop
           cmp.state, state.navigation, state.options, state.geoContext);
     }
 
-    Result<BoundState> boundState(
+    Result<typename SingleStepper::BoundState> boundState(
```

Comment:
Is this better than typedef'ing it up top?

Reply:
Hmm there's this single occurrence of this type in the MultiStepper class, not sure if this requires a typedef